lib/HLSL/HLOperationLower.cpp

```diff
@@ -6815,7 +6805,7 @@ static Value* ExtractFromTypedBufferLoad(const ResRetValueArray& ResRet,
   return ScalarizeElements(ResultTy, Elems, Builder);
 }
 
-Value *GenerateStructBufLd(Value *handle, Value *bufIdx, Value *offset,
+Value *GenerateRawBufLd(Value *handle, Value *bufIdx, Value *offset,
```

Comment:
This was really generating `rawBufferLoad` dxil op so changed the function name.